### PR TITLE
pipe, now with sound

### PR DESCRIPTION
**Pull request by Erlord**
The forward pipe operator improved, now with Mario Bros pipe sound. There might be a small performance hit when using it though :)

### CODE REVIEW

**Comment by KevinRansom** [comment]
Unfortunately there was a build error.  Have you considered making the sound configurable, some of my Scottish colleagues would undoubtedly want to hear the sound of bagpipes, I myself would love to hear Thunderstruck by AC/DC.

Certainly this PR shows a lot of promise ... we might also have to consider the cross platform implications.

**Comment by Erlord** [comment]
Ah too bad it failed, definitely not very thoroughly tested, bag pipes are
good, but lack the proper "swoosh" sound a pipe operator needs IMHO

EW

**Comment by jeroldhaas** [comment]
Mario pipe is more a "glug glug glug" sound, though, isn't it? :wink:

:+1: fun idea

**Comment by Erlord** [comment]
True, but there is so much symbolism/recognition in that sound.

EW

File: src/fsharp/FSharp.Core/FSharp.Core.fsproj [hunk 1]
```diff
@@ -196,32 +206,16 @@
     <Compile Include="..\..\assemblyinfo\assemblyinfo.FSharp.Core.dll.fs">
       <Link>assemblyinfo.FSharp.Core.dll.fs</Link>
     </Compile>
+    <None Include="mario_pipe.wav">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```
Comment (rojepp) [inline src/fsharp/FSharp.Core/FSharp.Core.fsproj]: Copy always? Puh lease.

**Pull request closed without merging**